ui/lib/config-ui/addon/components/messages/page/list.js

```diff
@@ -28,10 +30,15 @@ export default class MessagesList extends Component {
   @service namespace;
   @service pagination;
   @service('app-router') router;
+  @service api;
 
   @tracked showMaxMessageModal = false;
   @tracked messageToDelete = null;
 
+  isStartTimeAfterToday = (message) => {
+    return isAfter(message.startTime, timestamp.now());
+  };
```

Comment:
This was originally on the model but only used here so it was an easy one to move.